<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraMenu</class>
 <widget class="QWidget" name="CameraMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CameraMenu</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QGridLayout" name="grid_main">
     <property name="spacing">
      <number>0</number>
     </property>
     <item row="1" column="0">
      <layout class="QGridLayout" name="grid_button">
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="1" column="1">
        <widget class="QPushButton" name="pb_set">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string>set</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="pb_del">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string>del</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pb_add">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>35</height>
          </size>
         </property>
         <property name="text">
          <string>add</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <widget class="MenuTabWidget" name="tw_item">
       <property name="tabPosition">
        <enum>QTabWidget::North</enum>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="ItemListWidget" name="itm_list_1">
        <attribute name="title">
         <string>Tab 1</string>
        </attribute>
       </widget>
       <widget class="ItemListWidget" name="itm_list_2">
        <attribute name="title">
         <string>Tab 2</string>
        </attribute>
       </widget>
       <widget class="ItemListWidget" name="itm_list_3">
        <attribute name="title">
         <string>Tab 3</string>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ItemListWidget</class>
   <extends>QWidget</extends>
   <header>item_listwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MenuTabWidget</class>
   <extends>QTabWidget</extends>
   <header location="global">menu_tabwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
